### PR TITLE
Initial check-in of amppackager helm chart

### PR DESCRIPTION
**Pull request by banaag**
Initial PR for creating click-to-deploy deployer for amppackager.
This includes the helm chart for amppackager: https://github.com/ampproject/amppackager/tree/master/deploy/gcloud

### CODE REVIEW

File: k8s/amppackager/chart/amppackager/templates/amppackager_cert_consumer_template.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-cert-consumer-deployment
+  labels:
+    app: {{ .Release.Name }}-consumer
```
Comment (vcanaa) [inline k8s/amppackager/chart/amppackager/templates/amppackager_cert_consumer_template.yaml:6]: You need to add app.kubernetes.io labels pointing to the application. Example:
https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/cassandra/chart/cassandra/templates/manifests.yaml

Comment (banaag) [reply k8s/amppackager/chart/amppackager/templates/amppackager_cert_consumer_template.yaml:6]: Done.

**Comment by banaag** [comment]
Thanks for the suggestions. Will add the application manifest and schema.yaml in subsequent PRs, this PR is just dedicated to the helm chart for now.

File: k8s/amppackager/chart/amppackager/templates/amppackager_cert_consumer_template.yaml (hunk 1 above)
Comment (banaag) [reply k8s/amppackager/chart/amppackager/templates/amppackager_cert_consumer_template.yaml:6]: Done

**Pull request merged**